packages/xeus/src/worker.ts

```diff
@@ -261,16 +285,20 @@ export abstract class EmpackedXeusRemoteKernel extends XeusRemoteKernelBase {
       );
     }
 
-    await loadSharedLibs({
-      sharedLibs: this._sharedLibs,
-      prefix: '/',
-      Module: this.Module,
-      logger: this.logger
-    });
+    // Load shared libs only for old emscripten versions
+    if (this.emscriptenMajorVersion < 4) {
+      await loadSharedLibs({
+        sharedLibs: this._sharedLibs,
+        prefix: '/',
+        Module: this.Module,
+        logger: this.logger
+      });
+    }
 
```

Comment:
maybe we should throw an error if the emscripten-version is undefined (ie there was no abi package, which should not happen)

Reply:
Agreed, I added it already in https://github.com/jupyterlite/xeus/pull/311/files#diff-1455968df1146994ae6a5ff369ecf395c736a491d9112d80f35aeb1b745506c5R193